crates/bevy_render/src/base_bundle.rs

```diff
@@ -0,0 +1,59 @@
+use bevy_ecs::prelude::Bundle;
+use bevy_transform::prelude::{GlobalTransform, Transform};
+
+use crate::view::{ComputedVisibility, Visibility};
+
+/// A [`Bundle`] with the following [`Component`](bevy_ecs::component::Component)s:
+/// * [`Visibility`] and [`ComputedVisibility`], which describe the visibility of an entity
+/// * [`Transform`] and [`GlobalTransform`], which describe the position of an entity
+///
+/// * To show or hide an entity, you should set its [`Visibility`].
+/// * To get the computed visibility of an entity, you should get its [`ComputedVisibility`].
+/// * To place or move an entity, you should set its [`Transform`].
+/// * To get the global position of an entity, you should get its [`GlobalTransform`].
```

Comment:
Not just global position, but global transform. :) Maybe note that it is only valid for the current frame after transform propagation has been run?

Reply:
On the one hand, this is useful information. On the other hand, idk if we should be duplicating it across every bundle that includes GlobalTransform.

Reply:
I do agree that "global transform" is preferable to "global position".